packages/plugins/Gitcoin/src/SiteAdaptor/hooks/useProject.ts

```diff
@@ -1,17 +1,13 @@
-import { PluginID } from '@masknet/shared-base'
-import { useQuery } from '@tanstack/react-query'
 import { fetchApplications, fetchProjectById } from '../../apis/index.js'
+import { useAsyncRetry } from 'react-use'
 
 export function useProject(id: string) {
-    return useQuery({
-        queryKey: [PluginID.Gitcoin, 'project', 'id'],
```

Comment:
should be `[PluginID.Gitcoin, 'project', id]`

Reply:
This is the deleted code.